package.json

```diff
@@ -20,8 +20,8 @@
   },
   "scripts": {
     "test": "npm run lint && npm run style",
-    "style": "node_modules/.bin/jscs tests/tests.js",
-    "lint": "node_modules/.bin/jshint tests/tests.js"
+    "lint": "node node_modules/jshint/bin/jshint www && node node_modules/jshint/bin/jshint src && node node_modules/jshint/bin/jshint tests",
```

Comment:
Just a nitpick - you can omit path to jshint executable, since npm magic [does this for you](https://docs.npmjs.com/cli/run-script)